infra/compose/.env.example

```
@@ -1,14 +1,20 @@
-# Copy this file to infra/compose/.env before running docker compose or just.
+# Copy this file to infra/compose/.env before running production Compose commands.
+
 COMPOSE_PROJECT_NAME=nijigen-video-site
+
 JDK_VERSION=25
 API_PROD_RUNTIME_TARGET=api-native-runtime
 
+POSTGRES_VERSION=17
 POSTGRES_DB=nijigen_video_site
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_DB key should go before the POSTGRES_VERSION key

(UnorderedKey)